src/gwt/src/org/rstudio/studio/client/workbench/views/source/editors/text/ImagePreviewer.java

```diff
@@ -522,6 +553,29 @@ private static void onPreviewImage(DocDisplay display,
       panel.setPopupPosition(coordinates.getPageX(), coordinates.getPageY() + 20);
       panel.show();
    }
+   
+   private static final native void injectAttributes(Element el, String attributes)
+   /*-{
+      // create an element, inject inner HTML, and then extract attributes
+      var ctr = $doc.createElement("div");
+      ctr.innerHTML = "<div " + attributes + ">";
```

Comment:
I'm guessing that we should probably sanitize input here, since we're basically just stuffing arbitrary user input into `innerHTML`. Any thoughts on a better way to handle this, or an appropriate way to sanitize user input here?